src/components/SearchForm/SearchForm.js

```diff
@@ -3,7 +3,7 @@ import './SearchForm.css';
 function SearchForm({ onSearchClick }) {
     return (
         <form className="search__form" name='search-form' onSubmit={onSearchClick}>
-            <input className="search__input" type="text" id="search-input" name="search-input" placeholder="Enter topic" required />
+            <input className="search__input" type="text" id="search-input" name="searchInput" placeholder="Enter topic" required />
```

Comment:
any input should be controlled. 

To read more about `controlled components`: https://reactjs.org/docs/forms.html#controlled-components